api/src/main/java/org/regenstrief/linkage/io/DataBaseRecordStore.java

```diff
@@ -237,8 +249,17 @@ public boolean close() {
 			return true;
 		}
 		catch (SQLException sqle) {
-			sqle.printStackTrace();
-			return false;
+			throw new APIException(sqle);
 		}
 	}
+	
+	/**
+	 * Gets the value of isNewScratchTable
+	 *
+	 * @return the isNewScratchTable
+	 */
+	public boolean isNewScratchTable() {
+		return isNewScratchTable;
```

Comment:
Variable looks too similar to the method name that i would rather call it `newScratchTable`

Reply:
Done